roles/kubernetes/node/tasks/kubelet.yml

```
@@ -33,6 +33,7 @@
   template:
     src: "kubelet-config.{{ kubeadmConfig_api_version }}.yaml.j2"
     dest: "{{ kube_config_dir }}/kubelet-config.yaml"
+  notify: restart kubelet
```

I'm pretty sure It should be the full name (so in your case `Kubelet | restart kubelet`).

---

```suggestion
notify: Kubelet | restart kubelet
```
The indentation is also not correct.